cmd/minikube/cmd/config/config.go

```diff
@@ -257,6 +257,18 @@ var settings = []Setting{
 		name: "embed-certs",
 		set:  SetBool,
 	},
+	{
+		name: "no-native-ssh",
+		set: SetBool,
+	},
+	{
+		name: "mount",
+		set: SetBool,
+	},
+	{
+		name: "mount-string",
+		set: SetString,
+	},
```

Comment:
fail to see the connection between ssh and "mount" ?

Reply:
I could have added this to a second PR if you'd prefer.  They were just not added when the 'mount' and 'mount-string' flags were added and I needed them all in my use case.

Reply:
I moved it out and will open a separate PR.